docs/api/ledger.md

```diff
@@ -0,0 +1,46 @@
+# 💰 Ledger
+
+The `Ledger` object maintains a persistent ledger of all accounts and their
+balances of L2 tez (stored as mutez). Additionally, the `Ledger` object stores the 'self address' of
+the smart function, which is the address of the smart function itself.
+
+All operations on `Ledger` are synchronous and atomic, committed if the request to the smart function suceeds.
+
+## Quick Start
+
+We can obtain the balance of an account using `Ledger.balance()`:
+
+```typescript
+const alice: Address = "tz4abc...";
+console.log(Ledger.balance(alice)); // 0
+```
+
+The _self address_ of the smart function is accessible from the readonly property `Ledger.selfAddress`:
+
+```typescript
+console.log(Ledger.balance(Ledger.selfAddress)); // 420
+```
+
+Transfers are performed using `Ledger.transfer()`:
+
+```typescript
+Ledger.transfer(alice, 420); // Transfer 420 mutez to Alice from the balance of the smart function
+console.log(Ledger.balance(alice)); // 420
+console.log(Ledger.balance(Ledger.selfAddress)); // 0
+```
+
+## Instance Properties
+
+### `readonly Ledger.selfAddress: Address`
```

Comment:
Where will the definition of type Address be available?

Reply:
I will add `Address` to the `ledger.md` docs